src/test/resources/hw18NG.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="hw18NG">
+
+    <parameter name="email" value="Test321@gmail.com"/>
+    <parameter name="password" value="te$t$tudent"/>
+
+    <test name="Play a Song" preserve-order="true">
+        <classes>
+            <class name="Homework18"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223036583-6fb82b12-abf7-4ff0-aad8-38e4fa2dba78.png)